src/ex4/dist/style.css

```
@@ -0,0 +1,196 @@
+
+.background {
+  padding: 45px;
+  width: 100%;
+  height: 100vh;
+  background-image: linear-gradient(120deg, lightblue, rgb(45, 111, 133));
+}
+  input[type=text] {
+    padding-left: 15px;
+    width: 79%;
+    padding: 15px;
+    margin: 5px 0 22px 0;
+    border: none;
+    background: #f1f1f1;
+  }
+  
+  input[type=text]:focus {
+    background-color: #ddd;
+    outline: none;
+  }
+
+#footer {
+  display: inline; 
+  width: 80px;
+  height: 90px;
+}
+#footer #h3 {
```

h3 is an id? if it is you'll be able to use it only once i guess you ment
```suggestion
#footer h3 {
```